src/env.h

```diff
@@ -96,6 +96,8 @@ namespace node {
   V(fsevent_string, "FSEvent")                                                \
   V(gid_string, "gid")                                                        \
   V(handle_string, "handle")                                                  \
+  V(hasbyaddr, "hasByAddr")                                                   \
```

Comment:
to be more consistent, should this be `hasbyaddr_string` and `hasbyname_string`?